.github/workflows/publish.yml

```
@@ -12,15 +12,28 @@ jobs:
+          echo "npm (after): $(npm --version)"
+
       - name: Install dependencies
         run: npm ci
+
       - name: Build
         run: npm run build
-      - name: Publish package on NPM
+
+      - name: Publish package on NPM (OIDC)
         run: npm publish
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)